### ISSUE TITLE
Starting mattermost-desktop freezes other GUI windows

### ISSUE THREAD

**Issue by fcharras** [mattermost/desktop#2264]
I confirm (by marking "x" in the [ ] below: [x]):

- [x] This is not a troubleshooting question. [Troubleshooting questions go here: https://docs.mattermost.com/install/troubleshooting.html](https://docs.mattermost.com/install/troubleshooting.html).
- [x] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [x] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**

If some combination of desktop applications are running together, my GTK and QT applications start to show a weird behavior: the window stop refreshing until the mouse cursor moves. By "stop refreshing" I mean that if the window is active, and I scroll down, or use the keyboard to perform some actions, the window remain frozen (but the actions are registered, it's only the graphical side that is not updated), until I move the cursor, then the window will update.

Other applications are unaffected (e.g. chromium is not bugged).

The mattermost desktop app is not affected either.

**Environment**
- Operating System: Endeavour os (using the i3wn installer / I don't use any compositor / I set the arc-dark theme using the GUI shipped in endeavor os)
- Mattermost Desktop App version: 5.1.1-3 (installed with the official package in arch user repositories)
- Mattermost Server version: ireelevant ?

**Steps to reproduce**

Install the i3wm version of endeavour os. Install firefox, python-spyder IDE, thunderbird, mattermost-desktop packages. Start all the those applications one after another, in separate workspaces.

**Expected behavior**

Everything runs fine. In particular qt (spyder) and gtk (firefox, thunderbird) application windows are refreshing normally even if the cursor remains still. 

**Observed behavior**

Firefox, thunderbird and spyder (gtk and qt applications in general ?) are virtually unusable because the graphical windows do not refresh anymore unless the cursor move. Some other applications are not affected, like chromium. The only way out is to reboot and not start `mattermost-desktop` app (but using it in a firefox window is fine).

**Possible fixes**

I don't have an idea for a fix, but I'd like to add that the bug should be fairly easily reproducible, I didn't add much tweaks to endeavour os i3wn base installation.

**Comment by devinbinnie** [comment]
@fcharras This seems like an OS dependent one, since it's not something that reproduces on Ubuntu or any other distribution that I know of. I haven't tried pure Arch, but I know Manjaro works so I assume Endeavour should be fine.

Can you try installing an older version and see if the issue is resolved? I would be interested to see if it's an Electron issue, but my current guess would be that potentially the WM has issues, or some other component of the OS is causing a problem here.

**Comment by fcharras** [comment]
I've ran a few more experiments and it confirms that it's mattermost that is triggering the freeze of other applications and nothing else.

But in fact I've found that the bug also triggers when I open mattermost in the browser (in firefox or chromium) so I've opened the issue in the wrong place, sorry. Closing this one, I'll reopen it in the appropriate repo.

Endeavour OS is closer to vanilla arch than manjaro is (IIRC manjaro has custom package repositories and is lagging behind in versions).

Also if I had to guess I'd say that it must be related to something with i3wm. Also the issue started very recently (about 2 weeks ago).
